dbms/src/Flash/Planner/plans/PhysicalTableScan.cpp

```diff
@@ -66,12 +66,6 @@ void PhysicalTableScan::transformImpl(DAGPipeline & pipeline, Context & context,
     NamesWithAliases schema_project_cols;
     for (size_t i = 0; i < schema.size(); ++i)
     {
-        RUNTIME_CHECK(
-            schema[i].type->equals(*storage_schema[i].type),
-            schema[i].name,
-            schema[i].type->getName(),
-            storage_schema[i].name,
-            storage_schema[i].type->getName());
```

Comment:
the `storage_schema` is generated by `storage_interpreter`, and `storage_interpreter` is constructed by `tidb_table_scan`. But the `tidb_table_scan` come from tidb, which do not support `delmark_column`, so we can not revise `tidb_table_scan`, remove the check.